utilities/training_logger.py

```diff
@@ -19,24 +22,26 @@ def __init__(self, hyper_params, loss_freq=100, rollout_freq=1000):
         self.rollout_freq = rollout_freq
 
     def step(self, losses, rollout_batch, prediction):
-        """Perform a logging step: Update inner iteration counter and log info if needed
+        """Perform a logging step: update inner iteration counter and log info if needed.
```

Comment:
Maybe keep the caps after ":" or change all of them for consistency

Reply:
Ok